### PR TITLE
217936 create vulnerability details section for component scan

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/vulnerability-list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    )}
+  </List>
+);
+
+const VulnerabilityItem = ({
+  vulnerability,
+}: {
+  vulnerability: VulnerabilityModel;
+}): JSX.Element => (
+  <Card>
```
Comment (oleterno-equinor) [inline src/components/vulnerability-list/index.tsx:82]: Add class to use for styling
```suggestion
  <Card className="vulnerability-list">
```

File: src/components/vulnerability-list/style.css [hunk 2]
```diff
@@ -1,34 +1,38 @@
+.severity-list__item-description--markdown {
+  border: solid 1px var(--eds_text_static_icons_dark__secondary);
+  border-radius: 4px;
+  padding: var(--eds_spacing_small);
+  word-break: break-word;
+}
```
Comment (oleterno-equinor) [inline src/components/vulnerability-list/style.css:6]: Style links in description markdown.
```suggestion
}
.severity-list__item-description--markdown > ul > li > a {
  color: var(--eds_interactive_primary__resting);
  cursor: pointer;
}
```

File: src/components/vulnerability-list/style.css [hunk 3]
```diff
@@ -1,34 +1,38 @@
+.severity-list__item-description--markdown {
```
Comment (oleterno-equinor) [inline src/components/vulnerability-list/style.css:1]: Style cursor in vulnerability list
```suggestion
.vulnerability-list * {
  cursor: auto;
}
.severity-list__item-description--markdown {
```

File: src/components/vulnerability-list/index.tsx (hunk 1 above)
Comment (Pespiri) [reply src/components/vulnerability-list/index.tsx]: fixed by refactoring component

File: src/components/vulnerability-list/style.css (hunk 3 above)
Comment (Pespiri) [reply src/components/vulnerability-list/style.css:5]: fixed by refactoring component

File: src/components/vulnerability-list/style.css (hunk 2 above)
Comment (Pespiri) [reply src/components/vulnerability-list/style.css:10]: corrected

File: src/components/vulnerability-list/style.css [hunk 4]
```diff
@@ -1,34 +1,48 @@
+.severity-list__item {
+  margin-left: var(--eds_spacing_small);
+  margin-right: var(--eds_spacing_small);
```
Comment (oleterno-equinor) [inline src/components/vulnerability-list/style.css:3]: Can be achieved with one line. 
```suggestion
  margin: 0 var(--eds_spacing_small);
```

**Review by oleterno-equinor — APPROVED** [review]

**Pull request merged**